server/config/options.go

```diff
@@ -39,6 +40,13 @@ type AuthConfig struct {
 	LogOnly          bool
 }
 
+type CdcConfig struct {
```

Comment:
remove it from here? I think we want it to be in the outside config to avoid dependency issue?

Reply:
I was able to avoid a dependency issue by having `kv` no longer depend on `cdc`

Reply:
but with this server is importing the CDC package and the CDC package is importing the server package :) - maybe because it is in the config package that's why you're not seeing a dependency issue.